src/stories/Stack.stories.tsx

```diff
@@ -60,7 +60,7 @@ export const Nested: Story = {
 
 export const MarkupAsList: Story = {
   render: (args) => (
-    <Stack {...args} as="li">
+    <Stack {...args} as="ul">
```

Comment:
This is not related to the switch component issue, but it came up upgrading `markuplint` major version.